### ISSUE TITLE
Images should be click-to-show

### ISSUE THREAD

**Issue by aletson** [aletson/votefinder-web#44]
_proposed fix:_

in PageParser.ReadPostValues(): 

```python
for img in post.bodySoup.findAll("img"):
    image_url = img["src"]
    # add page element for deferred click-to-show with image_url in the same place in the document
    img.replaceWith('<div class="embedded-image not-loaded" data-image="'+image_url+'">Click to load image...</div>')
```

style.css:
```css
.not-loaded {
    font-size: 10px;
}
```

in game template:
```javascript
$(document).on("click", ".not-loaded", function() {
    $(this).html('<img src="'+$(this).data("image")+'" />').removeClass('.not-loaded');
});
```

- [x] Remove existing img tag strip code
- [x] Add Javascript to handle deferred click-to-show

**Comment by velinath** [comment]
This would make more sense ot do in BS4 but that's blocked by the LXML parser not encoding in UTF-8 on Python 2, meaning step 1 should be to migrate the app to Python 3. _Whee_.

**Comment by velinath** [comment]
It could be a problem with the way BS4 and lxml are interacting, it could be a Unicode thing. I tried forcing Unicode and it stripped out a ton of stuff, and I'm not sure why.

Maybe try [pyquery](https://pypi.python.org/pypi/pyquery) which uses lxml on the backend and see how it goes? Or try using UnicodeDammit the way that lxml recommends:
```python
from BeautifulSoup import UnicodeDammit

def decode_html(html_string):
     converted = UnicodeDammit(html_string, isHTML=True)
     if not converted.unicode:
         raise UnicodeDecodeError(
             "Failed to detect encoding, tried [%s]",
             ', '.join(converted.triedEncodings))
     # print converted.originalEncoding
     return converted.unicode

root = lxml.html.fromstring(decode_html(tag_soup))
```

**Comment by velinath** [comment]
#50

**Comment by aletson** [comment]
This is no longer blocked.

**Comment by aletson** [comment]
L255: 
`[img.replaceWith('<div class="embedded-image not-loaded" data-image="'+img['url']+'">Click to load image...</div>') for img in post.bodySoup.find_all("img")]`

**Comment by aletson** [comment]
needs tested in beta before next release

**Comment by aletson** [comment]
weird things are happening when images are nested in urls but also it's not, generally speaking, working, so more effort needs be made yet

**Comment by aletson** [comment]
Issues are all worked out in `master`, posts moving forward after next release will embed images
